test/buildable/default.nix

```diff
@@ -19,7 +19,7 @@ in recurseIntoAttrs {
     buildCommand = 
       (concatStrings (mapAttrsToList (name: value: ''
         printf "checking whether executable runs... " >& 2
-        cat ${value.run}
+        cat ${haskellLib.check value}
```

Comment:
I think it's drawing a long bow to use the `haskellLib.check` function here, for something that isn't a haskell test suite.
What about simply using `${value}/${value.installedExe}` ?

Reply:
I am a little reluctant to change that because:

* It would need to include `${value.config.testWrapper}` as well for windows (not so bad here but tricky when `value` is a long path.
* It would run the tests in the same derivation (so less scope to run them in parallel)

I don't think there is anything in `check` that is specific to test suites (vs exe components), but perhaps we could add a similar `run` function for use with `exes`?